syft/pkg/cataloger/package_exclusions.go

```diff
@@ -0,0 +1,68 @@
+package cataloger
+
+import (
+	"golang.org/x/exp/slices"
+
+	"github.com/anchore/syft/syft/artifact"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/pkg/cataloger/alpm"
+	"github.com/anchore/syft/syft/pkg/cataloger/apkdb"
+	"github.com/anchore/syft/syft/pkg/cataloger/binary"
+	"github.com/anchore/syft/syft/pkg/cataloger/deb"
+	"github.com/anchore/syft/syft/pkg/cataloger/nix"
+)
+
+type CategoryType string
+
+const (
+	OsCatalogerType     CategoryType = "os"
+	BinaryCatalogerType CategoryType = "binary"
+)
+
+var CatalogerTypeIndex = map[CategoryType][]string{
+	"os": {
```

Comment:
nit: could use the constants (`OsCatalogerType` and `BinaryCatalogerType`) from above for the keys here

Reply:
👍 resolved with [14263d3](https://github.com/anchore/syft/pull/1948/commits/14263d3d6efe77da9989aaf523462e730ca6718b)